src/Psphere.jl

```diff
@@ -62,6 +62,18 @@ function psphere_eval(psphere::AbstractVector{<:Real}, super::Supercell, site_li
     return a
 end
 
+function psphere_eval(psphere::AbstractVector{<:Real}, super::Supercell, site::AbstractVector{AbstractVector{<:Real}})
```

Comment:
If there's a lot of shared logic between the two methods, it might be a good idea to call one method from the other.

Reply:
Consolidated into one function https://github.com/xamberl/DFT-raMO/blob/b84386058f905f6464147f1892c067fe770028c2/src/Psphere.jl#L53